doc/neps/nep-0045-c_style_guide.rst

```diff
@@ -0,0 +1,269 @@
+=================================
+NEP 45 — C Style Guide
+=================================
+
+:Author: Charles Harris <charlesr.harris@gmail.com>
+:Author: Pierre de Buyl <pdebuyl@pdebuyl.be>
+:Author: Matti Picus <matti.picus@gmail.com>
+:Author: Warren Weckesser <warren.weckesser@gmail.com>
+:Author: Parul Aggarwal <parulagg27@gmail.com>
+:Status: Draft
```

Comment:
I think this should be Accepted. It's not a new proposal that should go through review. It's our existing process, changes to it are out of scope for this PR.